pkg/apiserver/interfaces.go

```diff
@@ -101,5 +101,6 @@ type ResourceWatcher interface {
 // Redirector know how to return a remote resource's location.
 type Redirector interface {
 	// ResourceLocation should return the remote location of the given resource, or an error.
+	// shouldn't return `"", nil`
```

Comment:
nit: remove the comment. I think its redundant and hard to parse without the context.